api/src/pcapi/connectors/api_adresse.py

```diff
@@ -51,6 +57,18 @@ class AddressInfo(pydantic_v1.BaseModel):
     city: str
     street: str | None
 
+    @pydantic_v1.validator("latitude")
+    def validate_latitude(cls, latitude: float) -> float:
+        if not -MAX_LATITUDE < latitude < MAX_LATITUDE:
+            raise ValueError("latitude out of bounds")
+        return latitude
+
+    @pydantic_v1.validator("longitude")
+    def validate_longitude(cls, longitude: float) -> float:
+        if not -MAX_LONGITUDE < longitude < MAX_LONGITUDE:
```

Comment:
Détail : pour couvrir partout, on devrait accepter la longitude 180° : 
```suggestion
        if not -MAX_LONGITUDE < longitude <= MAX_LONGITUDE:
```
mais pour le cas particulier du pass Culture, on n'a aucun territoire en France sur ce méridien, on n'aura donc pas d'offre aux îles Fidji ici : https://fr.wikipedia.org/wiki/180e_m%C3%A9ridien#/maplink/11 (le reste est dans la Mer).

De même pour le pôle Nord et le pôle Sud concernant la latitude (+90 et -90°)
Donc même si tu ne couvres pas toute la planète, ça passera !  :slightly_smiling_face: